docs/manual/build_system/cmake_variables.qbk

```diff
@@ -66,6 +67,7 @@ The options are split into these categories:
 [variablelist
         [[[#build_system.cmake_variables.HPX_WITH_ASYNC_FUNCTION_COMPATIBILITY] `HPX_WITH_ASYNC_FUNCTION_COMPATIBILITY:BOOL`][Enable old style ..._sync/..._async functions in API (default: OFF)]]
         [[[#build_system.cmake_variables.HPX_WITH_AUTOMATIC_SERIALIZATION_REGISTRATION] `HPX_WITH_AUTOMATIC_SERIALIZATION_REGISTRATION:BOOL`][Use automatic serialization registration for actions and functions. This affects compatibility between HPX applications compiled with different compilers (default ON)]]
+        [[[#build_system.cmake_variables.HPX_WITH_DISABLED_SIGNAL_EXCEPTION_HANDLERS] `HPX_WITH_DISABLED_SIGNAL_EXCEPTION_HANDLERS:BOOL`][Disables the mechanism that produces debug output for signals and unhandled exceptions (default: OFF)]]
         [[[#build_system.cmake_variables.HPX_WITH_BENCHMARK_SCRIPTS_PATH] `HPX_WITH_BENCHMARK_SCRIPTS_PATH:PATH`][Directory to place batch scripts in]]
```

Comment:
This line should get automatically generated, so it will have the same descriptive text as specified in the cmake file. No need to manually edit those.